### ISSUE TITLE
GT Screen-Scraper's error: 0x80073CF6

### ISSUE THREAD

**Issue by PureOcean** [SubtitleEdit/subtitleedit#3842]
GT Screen-Scraper give the following error:

> Error Message:
> Package could not be registered. (Exception from HRESULT: 0x80073CF6)
> 
> Stack Trace:
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
>    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    at Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlProcess.<CreateWebViewControlHostAsync>d__16.MoveNext()
> --- End of stack trace from previous location where exception was thrown ---
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
>    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
>    at Microsoft.Toolkit.Forms.UI.Controls.WebViewControlProcessExtensions.CreateWebViewControlHost(WebViewControlProcess process, IntPtr hostWindowHandle, Rectangle bounds)
>    at Microsoft.Toolkit.Forms.UI.Controls.WebView.Initialize()
>    at Microsoft.Toolkit.Forms.UI.Controls.WebView.System.ComponentModel.ISupportInitialize.EndInit()
>    at WebViewTranslate.MainForm.buttonTranslate_Click(Object sender, EventArgs e)
>    at System.Windows.Forms.Control.OnClick(EventArgs e)
>    at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
>    at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
>    at System.Windows.Forms.Control.WndProc(Message& m)
>    at System.Windows.Forms.ButtonBase.WndProc(Message& m)
>    at System.Windows.Forms.Button.WndProc(Message& m)
>    at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

I think, the problem cause by **Microsoft.Win32WebViewHost_cw5n1h2txyewy** in c:\Windows\SystemApps. 

The **Microsoft-Windows-AppXDeploymentServer/Operational** logs of the Event Viewer:

> Could not find the path C:\Windows\SystemApps\Microsoft.Win32WebViewHost_cw5n1h2txyewy\AppxManifest.xml in the State Repository database.
> 
> Creating Resiliency File C:\ProgramData\Microsoft\Windows\AppRepository\1d8450b5-b038-41ca-9519-1bc09c098532_S-1-5-21-2134135992-888563254-2092761094-1000_1.rslc for Register Operation on Package Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy.
> 
> Deployment Register operation on package Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy has been de-queued and is running for user SID S-1-5-21-2134135992-888563254-2092761094-1000.
> 
> Successfully added the following uri(s) to be processed:  C:\Windows\SystemApps\Microsoft.Win32WebViewHost_cw5n1h2txyewy\AppxManifest.xml.
> 
> Successfully added the following uri(s) to be processed:  C:\Windows\SystemApps\Microsoft.Win32WebViewHost_cw5n1h2txyewy\AppxManifest.xml.
> 
> error 0x80073B1F: Cannot register the Microsoft.Win32WebViewHost_cw5n1h2txyewy package due to the following error: ResourceMap Not Found.
> 
> Deployment Register operation with target volume C: on Package Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy from:  (AppxManifest.xml)  failed with error 0x80073CF6. See http://go.microsoft.com/fwlink/?LinkId=235160 for help diagnosing app deployment issues.
> 
> The last successful state reached was MutablePackageDirectoryAdd. Failure occurred before reaching the next state MachineRegisterAdd. hr: 0x80073CF6
> 
> AppX Deployment operation failed for package Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy with error 0x80073CF6. The specific error text for this failure is: error 0x80073B1F: Cannot register the Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy package because the following error was encountered: ResourceMap Bulunamadı.
> . Verify that the package's 'resources.pri' file in the package is valid.

I've trimmed needless Apps from Windows, probably because of it. 
I tried this command for re-register to **Microsoft.Win32WebViewHost**:

`PowerShell -ExecutionPolicy Unrestricted -Command "Add-AppxPackage -DisableDevelopmentMode -Register C:\Windows\SystemApps\Microsoft.Win32WebViewHost_cw5n1h2txyewy\AppxManifest.xml"`

The error:

> Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF6, Package could not be registered.
> error 0x80073B1F: Cannot register the Microsoft.Win32WebViewHost_10.0.18362.449_neutral_neutral_cw5n1h2txyewy package b
> ecause the following error was encountered: ResourceMap Not Found.

Is it possible to solve this problem without reinstalling windows?

Info: Subtitle Edit 3.5.11.99, GT Screen-Scraper 0.7, Windows 10.0.18362.476

**Comment by niksedk** [comment]
GT Screen-Scraper requires Win 10 10.0.17110.0 or later... but it seems like you have that. 
Do you also have .NET Framework 4.6.2 or a later release + the Edge browser installed?

As the next version of Edge (15th Jan) will be based on Chromium, perhaps it will be easier to use Chromium directly, but that will make the plugin very large (like 100mb).

**Comment by ilkertezcan** [comment]
Thank you for your reply.

Interestingly, no problem **Bing screen-scraping** plugin.

I don't install .NET Framework 3.5.
Already installed the lastest version of .NET Framework

> HKLM\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full
> Release: 528040
> 

Microsoft Edge is broken. I just repaired this commands:

On CMD: 
`RD /s /q "%localappdata%\Packages\Microsoft.MicrosoftEdge_8wekyb3d8bbwe"`

On Powershell: 
`Get-AppXPackage -AllUsers -Name Microsoft.MicrosoftEdge | Foreach {Add-AppxPackage -DisableDevelopmentMode -Register “$($_.InstallLocation)\AppXManifest.xml” -Verbose}`

I set up the M.Edge as default browser (because it wasn't). I tried GT Screen-scraper again  in **SubtitleEdit**. It give same error.

You're right. As of M.Edge moving to Chromium full-fledged, we will avoid these and similar problems. Hooray the **Win32-app** world.
